project/Dependencies.scala

```diff
@@ -97,7 +97,7 @@ object Dependencies {
       filters,
       ws,
       // We can't update this to 4.0.0 due to an incompatibility with Play 2.8.x, attempt to update along with Play
-      "com.typesafe.akka" %% "akka-testkit" % "2.6.20" % Test, // scala-steward:off
```

Comment:
Remove the Scala Steward config for this dependency as we're addressing this upstream. See https://github.com/guardian/scala-steward-public-repos/pull/54.